src/snowflake/snowpark/mock/_stage_registry.py

```diff
@@ -116,6 +117,16 @@ def extract_stage_name_and_prefix(stage_location: str) -> Tuple[str, str]:
     return stage_name, dir_path
 
 
+def copy_files_and_dirs(src, dst):
+    try:
+        shutil.copytree(src, dst)
```

Comment:
Can we use `os.path.isfile` and `os.path.isdir` instead?

Reply:
yes, code updated. easier to understand now